src/AblyBroadcaster.php

```diff
@@ -49,9 +57,12 @@ class AblyBroadcaster extends Broadcaster
     public function __construct(AblyRest $ably, $config)
     {
         $this->ably = $ably;
-        if (self::$serverTimeDiff == null) {
-            self::setServerTime(round($this->ably->time() / 1000));
-        }
+
+        // Local file cache is preferred to avoid sharing serverTimeDiff across different servers
+        $this->serverTimeDiff = Cache::store('file')->remember('ably_server_time_diff', 86400, function() {
```

Comment:
Maybe a timeout can be made `6 hr => 360 mins => 21600 seconds`.
 During server time glitch, this can correct the difference as early as possible. There's really not much overhead ...